src/wifi/model/mac-low.h

```diff
@@ -1147,11 +1147,14 @@ double mpduSnr;
     Ptr<WifiMacQueue> aggregateQueue;
   } AllocationParameters;
 
-  typedef std::map<AllocationID, AllocationParameters> AllocationPeriodsTable;
+  typedef std::pair<Mac48Address, Mac48Address> AddressPair;
+  typedef std::map<AddressPair, AllocationParameters> AllocationPeriodsTable;
```

Comment:
are the dst and src addesses enough? why isn't allocation ID included as well?

Reply:
Allocation ID is not useful. In the case of a broadcast CBAP whose allocation ID is 0, if a suspended transmission is stored with (ID, sourceMAC, destMAC) then how is this transmission smoothly resumed in a hypothetical successive SP between the two STAs, which has to have allocation ID different from 0? We should discuss, I'm not saying my solution is the best one. Probably the best solution is to completely re-write these parts of code